client/app/components/ParameterMappingInput.jsx

```diff
@@ -149,7 +150,7 @@ export class ParameterMappingInput extends React.Component {
       <div className="m-t-10">
         <Select
           className="w-100"
-          defaultValue={mapping.mapTo}
+          value={mapping.mapTo}
```

Comment:
This is so that if edit is cancelled (making `state.mapping` reset to original `props.mapping`), it would affect these input components, or else they would have stayed as left.